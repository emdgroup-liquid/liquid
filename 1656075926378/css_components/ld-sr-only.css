.ld-sr-only,:host{clip:rect(0,0,0,0);border-width:0;height:var(--ld-sp-1);overflow:hidden;padding:0;position:absolute;width:var(--ld-sp-1)}