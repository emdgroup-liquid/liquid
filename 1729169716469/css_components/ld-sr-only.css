.ld-sr-only,:host{height:var(--ld-sp-1);overflow:hidden;padding:0;position:absolute;width:var(--ld-sp-1);clip:rect(0,0,0,0);border-width:0}