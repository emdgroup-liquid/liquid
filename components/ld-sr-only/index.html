
<!doctype html>
<html lang="en">
<head>
  <base href="/liquid/">
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Liquid â€“ Screen Reader Only</title>

  <link rel="apple-touch-icon" sizes="180x180" href="/liquid/assets/apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/liquid/assets/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/liquid/assets/favicon-16x16.png">
  <link rel="manifest" href="/liquid/assets/site.webmanifest">
  <link rel="mask-icon" href="/liquid/assets/safari-pinned-tab.svg" color="#eb3c96">
  <meta name="msapplication-TileColor" content="#eb3c96">
  <meta name="theme-color" content="#ffffff">

  <script>
    __docsSearchIndex__ = [{
              title: 'CSS vs. Web Components',
              tags: '',
              url: '/liquid/liquid/components-assets/',
              headings: 'Tweaking the bundler and adjusting the assets path',
              breadcrumbs: [
                
                'Liquid','CSS vs. Web Components']
            },{
              title: 'Components',
              tags: '',
              url: '/liquid/components/',
              headings: 'Status',
              breadcrumbs: [
                
                ]
            },{
              title: 'Button',
              tags: '',
              url: '/liquid/components/ld-button/',
              headings: 'ExamplesPrimaryDisabledSecondaryOn brand colorSecondary on brand colorGhostHighlightDangerSizeWith iconWith custom widthJustify contentMulti-lineAnchor buttonProperties',
              breadcrumbs: [
                
                'Components','Button']
            },{
              title: 'Checkbox',
              tags: '',
              url: '/liquid/components/ld-checkbox/',
              headings: 'ExamplesPrimaryDisabledDarkHighlightInvalidDangerWith labelWith label and input messageInput validationPropertiesDependenciesUsed byGraph',
              breadcrumbs: [
                
                'Components','Checkbox']
            },{
              title: 'Heading',
              tags: '',
              url: '/liquid/components/ld-heading/',
              headings: 'ExamplesWith heading levelWith deviating visual level b*With deviating visual level xh*With deviating visual level xb*Heading colorsProperties',
              breadcrumbs: [
                
                'Components','Heading']
            },{
              title: 'Icon',
              tags: '',
              url: '/liquid/components/ld-icon/',
              headings: 'Web ComponentDifferent sizesColorWith custom SVG iconPropertiesSlots',
              breadcrumbs: [
                
                'Components','Icon']
            },{
              title: 'Input Message',
              tags: '',
              url: '/liquid/components/ld-input-message/',
              headings: 'ExamplesAs error messageAs info messageAs success messageProperties',
              breadcrumbs: [
                
                'Components','Input Message']
            },{
              title: 'Input',
              tags: '',
              url: '/liquid/components/ld-input/',
              headings: 'ExamplesDefaultDisabledDarkInvalidType dateType emailType fileType numberType passwordType searchType telType timeType urlMultiline (as textarea)SizeWith labelWith label and input messageWith slotted elementsInput validationPropertiesSlots',
              breadcrumbs: [
                
                'Components','Input']
            },{
              title: 'Label',
              tags: '',
              url: '/liquid/components/ld-label/',
              headings: 'ExampleSizePositionProperties',
              breadcrumbs: [
                
                'Components','Label']
            },{
              title: 'Paragraph',
              tags: '',
              url: '/liquid/components/ld-paragraph/',
              headings: 'ExamplesDefault paragraphSizesProperties',
              breadcrumbs: [
                
                'Components','Paragraph']
            },{
              title: 'Radio Button',
              tags: '',
              url: '/liquid/components/ld-radio/',
              headings: 'ExamplesPrimaryDisabledDarkHighlightInvalidDangerWith labelWith label and input messageInput validationProperties',
              breadcrumbs: [
                
                'Components','Radio Button']
            },{
              title: 'Select',
              tags: '',
              url: '/liquid/components/ld-select/',
              headings: 'ExamplesSingle select modeMultiple select modeDisabledInvalidDetachedInlineGhostSizeWith custom trigger button iconWith labelWith label and input messageInput validationPropertiesEventsSlots',
              breadcrumbs: [
                
                'Components','Select']
            },{
              title: 'Screen Reader Live',
              tags: '',
              url: '/liquid/components/ld-sr-live/',
              headings: 'How it worksExampleProperties',
              breadcrumbs: [
                
                'Components','Screen Reader Live']
            },{
              title: 'Screen Reader Only',
              tags: '',
              url: '/liquid/components/ld-sr-only/',
              headings: 'ExamplesProperties',
              breadcrumbs: [
                
                'Components','Screen Reader Only']
            },{
              title: 'Theme',
              tags: '',
              url: '/liquid/components/ld-theme/',
              headings: 'How to apply a themeTheme inceptionProperties',
              breadcrumbs: [
                
                'Components','Theme']
            },{
              title: 'CSS vs. Web Components',
              tags: '',
              url: '/liquid/liquid/css-vs-web-components/',
              headings: 'When to use what',
              breadcrumbs: [
                
                'Liquid','CSS vs. Web Components']
            },{
              title: 'Getting started',
              tags: '',
              url: '/liquid/liquid/getting-started/',
              headings: 'Import and register Web Componentsself-lazy-loading bundlecustom-elements bundleImport stylesheets',
              breadcrumbs: [
                
                'Liquid','Getting started']
            },{
              title: 'Globals',
              tags: '',
              url: '/liquid/global/',
              headings: '',
              breadcrumbs: [
                
                ]
            },{
              title: 'Border-radius',
              tags: '',
              url: '/liquid/global/border-radius/',
              headings: '',
              breadcrumbs: [
                
                'Globals','Border-radius']
            },{
              title: 'Colors',
              tags: '',
              url: '/liquid/global/colors/',
              headings: 'White and Liquid BlackGrey scale colorsBackground grey scale colorsForeground grey scale colorsRich colorsRich BlueRich GreenRich PurpleRich RedVibrant colorsVibrant CyanVibrant GreenVibrant MagentaVibrant YellowTransparent colorsTheme palettesBubblegumOceanShakeSolventTea',
              breadcrumbs: [
                
                'Globals','Colors']
            },{
              title: 'Focus',
              tags: '',
              url: '/liquid/global/focus/',
              headings: '',
              breadcrumbs: [
                
                'Globals','Focus']
            },{
              title: 'Fonts',
              tags: '',
              url: '/liquid/global/fonts/',
              headings: '',
              breadcrumbs: [
                
                'Globals','Fonts']
            },{
              title: 'Shadows',
              tags: '',
              url: '/liquid/global/shadows/',
              headings: 'Box shadowsActiveHoverStackedStickyDrop-shadows (CSS filter)ActiveHoverStackedSticky',
              breadcrumbs: [
                
                'Globals','Shadows']
            },{
              title: 'Spacings',
              tags: '',
              url: '/liquid/global/spacings/',
              headings: '',
              breadcrumbs: [
                
                'Globals','Spacings']
            },{
              title: 'Typography',
              tags: '',
              url: '/liquid/global/typography/',
              headings: 'Available CSS custom properties',
              breadcrumbs: [
                
                'Globals','Typography']
            },{
              title: 'Server-side rendering',
              tags: '',
              url: '/liquid/liquid/server-side-rendering/',
              headings: 'Known issues and workarounds',
              breadcrumbs: [
                
                'Liquid','Server-side rendering']
            },{
              title: 'Type checking and intellisense',
              tags: '',
              url: '/liquid/liquid/type-checking-and-intellisense/',
              headings: 'With JSXWith vanilla HTML',
              breadcrumbs: [
                
                'Liquid','Type checking and intellisense']
            }
    ]
  </script>

  <link rel="stylesheet" href="/liquid/docs.css">
  <link rel="stylesheet" href="/liquid/dist/build/liquid.css">
  <script type="module" src="/liquid/dist/build/liquid.esm.js"></script>
  <script nomodule src="/liquid/dist/build/liquid.js"></script>
</head>
<body>
  <script>
    if (/Safari/.test(navigator.userAgent) && /Apple Computer/.test(navigator.vendor)) {
      document.body.classList.add('sarari')
    }

    var storedThemePref = window.localStorage.getItem('docs-theme')
    var isDark = storedThemePref === 'dark' ? true : false
    document.body.classList.add(`docs-theme-${this.isDark ? 'dark' : 'light'}`)
    document.body.classList.remove(
      `docs-theme-${!this.isDark ? 'dark' : 'light'}`
    )
  </script>
  <docs-layout id="docs-layout" style="visibility: hidden">
    <docs-topbar></docs-topbar>

    <docs-nav>
      
      

      <ul>
          <li class="docs-nav__li docs-nav__li--active"><details>
                <summary class="docs-nav__summary docs-nav__summary--liquid">
                  <a href="/liquid/"><span aria-label="Liquid">Liquid</span></a>
                </summary>
                <ul>
          <li class="docs-nav__li"><a href="/liquid/liquid/getting-started/">Getting started</a></li>
          <li class="docs-nav__li"><a href="/liquid/liquid/css-vs-web-components/">CSS vs. Web Components</a></li>
          <li class="docs-nav__li"><a href="/liquid/liquid/components-assets/">Component assets</a></li>
          <li class="docs-nav__li"><a href="/liquid/liquid/type-checking-and-intellisense/">Type checking and intellisense</a></li>
          <li class="docs-nav__li"><a href="/liquid/liquid/server-side-rendering/">Server-side rendering</a></li></ul>
              </details></li>
          <li class="docs-nav__li"><details>
                <summary class="docs-nav__summary docs-nav__summary--globals">
                  <a href="/liquid/global/"><span aria-label="Globals">Globals</span></a>
                </summary>
                <ul>
          <li class="docs-nav__li"><a href="/liquid/global/border-radius/">Border-radius</a></li>
          <li class="docs-nav__li"><a href="/liquid/global/colors/">Colors</a></li>
          <li class="docs-nav__li"><a href="/liquid/global/focus/">Focus</a></li>
          <li class="docs-nav__li"><a href="/liquid/global/fonts/">Fonts</a></li>
          <li class="docs-nav__li"><a href="/liquid/global/shadows/">Shadows</a></li>
          <li class="docs-nav__li"><a href="/liquid/global/spacings/">Spacings</a></li>
          <li class="docs-nav__li"><a href="/liquid/global/typography/">Typography</a></li></ul>
              </details></li>
          <li class="docs-nav__li docs-nav__li--active"><details open>
                <summary class="docs-nav__summary docs-nav__summary--components">
                  <a href="/liquid/components/"><span aria-label="Components">Components</span></a>
                </summary>
                <ul>
          <li class="docs-nav__li"><a href="/liquid/components/ld-button/">Button</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-checkbox/">Checkbox</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-heading/">Heading</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-icon/">Icon</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-input/">Input</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-input-message/">Input Message</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-label/">Label</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-paragraph/">Paragraph</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-radio/">Radio Button</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-sr-live/">Screen Reader Live</a></li>
          <li class="docs-nav__li docs-nav__li--active"><a href="/liquid/components/ld-sr-only/">Screen Reader Only</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-select/">Select</a></li>
          <li class="docs-nav__li"><a href="/liquid/components/ld-theme/">Theme</a></li></ul>
              </details></li></ul>
    </docs-nav>

    <docs-nav-backdrop></docs-nav-backdrop>

    <docs-btn-toggle-nav></docs-btn-toggle-nav>

    
      <docs-toc style="order: 2;">
        <nav class="docs-toc__nav">
        <ol><li><a href="#examples">Examples</a></li><li><a href="#properties">Properties</a></li></ol></nav>
      </docs-toc>
    

    <docs-main>
      <header><docs-breadcrumbs>
            
            <li>
                <a href="/liquid/components/">Components</a>
              </li><li>
              <a href="/liquid/components/ld-sr-only/" aria-current="page">Screen Reader Only</a>
            </li>
          </docs-breadcrumbs><docs-edit-on-github path="./src/liquid/components/ld-sr-only/readme.md"></docs-edit-on-github>
      </header>
      <h1 id="ld-sr-only">ld-sr-only <a class="header-anchor" href="#ld-sr-only"><span aria-label="" class="header-anchor__symbol">#</span></a></h1>
<p>Use <code>ld-sr-only</code> to hide an element visually without hiding it from screen readers.</p>
<p>The CSS class <code>ld-sr-only</code> works the same way as its Web Component counterpart, except that it can be applied to an HTML element directly while the Web Component wraps the element which needs to be only screen reader visible.</p>
<hr>
<h2 id="examples">Examples <a class="header-anchor" href="#examples"><span aria-label="" class="header-anchor__symbol">#</span></a></h2>
<docs-example code="%3Cld-sr-only%3EHello%20screen%20reader%3C%2Fld-sr-only%3E" code-css-component="%3Cspan%20class%3D%22ld-sr-only%22%3EHello%20screen%20reader%3C%2Fspan%3E"  opened themable>
<div slot="code">
<pre class="language-html"><code class="language-html"><span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>ld-sr-only</span><span class="token punctuation">></span></span>Hello screen reader<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>ld-sr-only</span><span class="token punctuation">></span></span></code></pre>
</div><div slot="codeCssComponent">
<pre class="language-html"><code class="language-html"><span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>span</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>ld-sr-only<span class="token punctuation">"</span></span><span class="token punctuation">></span></span>Hello screen reader<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>span</span><span class="token punctuation">></span></span></code></pre>
</div><div slot="show"><ld-sr-only>Hello screen reader</ld-sr-only></div><div slot="showCssComponent"><span class="ld-sr-only">Hello screen reader</span></div></docs-example>
<!-- Auto Generated Below -->
<h2 id="properties">Properties <a class="header-anchor" href="#properties"><span aria-label="" class="header-anchor__symbol">#</span></a></h2>
<table>
<thead>
<tr>
<th>Property</th>
<th>Attribute</th>
<th>Description</th>
<th>Type</th>
<th>Default</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>key</code></td>
<td><code>key</code></td>
<td>for tracking the node's identity when working with lists</td>
<td><code>string \| number</code></td>
<td><code>undefined</code></td>
</tr>
<tr>
<td><code>ref</code></td>
<td><code>ref</code></td>
<td>reference to component</td>
<td><code>any</code></td>
<td><code>undefined</code></td>
</tr>
</tbody>
</table>
<hr>
<p><em>Built with <a href="https://stenciljs.com/">StencilJS</a></em></p>

    </docs-main>

  </docs-layout>
  <docs-search></docs-search>
</body>
</html>
