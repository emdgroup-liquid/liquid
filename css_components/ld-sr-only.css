.ld-sr-only,:host{position:absolute;width:var(--ld-sp-1);height:var(--ld-sp-1);padding:0;overflow:hidden;clip:rect(0,0,0,0);border-width:0}