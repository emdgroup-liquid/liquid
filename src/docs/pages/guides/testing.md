---
eleventyNavigation:
  key: Testing
  parent: Guides
  order: 10
layout: layout.njk
title: Testing
permalink: guides/testing/
---

# Testing

If you want to __unit test__ code in your app, which depends on the functionality of Liquid Oxygen Web Components, without mocking their internals, then you must make sure all relevant components have _hydrated_ in your test setup, before you execute your test code. You can do so by running the `defineCustomElements()` method imported from `'@emdgroup-liquid/liquid/dist/loader'` before each test and a `await`ing a helper function, which makes sure all components involved in the test have hydrated.

The following example shows how to execute unit tests with hydrated Web Components with [vitest](https://vitest.dev/), React, and the [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/).

First, [configure vitest to use a setup file](https://vitest.dev/config/#setupfiles):

```js
// vite.config.ts
import { defineConfig } from 'vitest/config'
export default defineConfig({
  // ...
  test: {
    // ...
    setupFiles: './src/setupTests.ts',
  },
})
```

The `setupTests.ts` file looks like this:

```js
// src/setupTests.ts
import { defineCustomElements } from '@emdgroup-liquid/liquid/dist/loader'
import '@testing-library/jest-dom'
import matchers from '@testing-library/jest-dom/matchers'
import { cleanup } from '@testing-library/react'
import { vi } from 'vitest'

// Extend Vitest's expect method with methods from react-testing-library.
expect.extend(matchers)

beforeAll(() => {
  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
  // @ts-ignore
  global.__LD_ASSET_PATH__ = '/' // removes CDN warning
  defineCustomElements()

  global.fetch = vi
    .fn()
    .mockReturnValue(Promise.resolve({ text: () => Promise.resolve('') }))
})

// Run a cleanup after each test case.
afterEach(() => {
  cleanup()
})
```

In your test file, you can now wait for the Web Components to hydrate before executing your actual test code.

```tsx
// src/App.test.tsx
import App from './App'
import { render, screen, waitFor, fireEvent } from '@testing-library/react'
import userEvent from '@testing-library/user-event'
import React from 'react'

async function waitForLiquidToHydrate() {
  return waitFor(() => {
    Array.from(document.querySelectorAll('*'))
      .filter((e) => e.tagName.startsWith('LD-'))
      .forEach((component) => {
        expect(component).toHaveClass('hydrated')
      })
  })
}

test('validates input', async () => {
  render(<App />)

  await waitForLiquidToHydrate()

  const ldInputName = screen.getByPlaceholderText('e.g. Jason Parse')
  expect(ldInputName).toBeInTheDocument()

  const ldButtonSubmit = screen.getByText('Submit')
  expect(ldButtonSubmit).toBeInTheDocument()
  userEvent.click(ldButtonSubmit)

  const form = await screen.findByTestId('form')
  expect(form).toBeInTheDocument()

  fireEvent.submit(form)

  await screen.findByText('Your full name is required.')
})
```

Check out our [React sandbox app](https://stackblitz.com/github/emdgroup-liquid/liquid-sandbox-react-tailwind?file=src%2FApp.test.tsx) for a full working example.

<docs-page-nav prev-href="guides/design-tokens/" next-title="Sandbox applications" next-href="guides/sandbox-applications/"></docs-page-nav>

