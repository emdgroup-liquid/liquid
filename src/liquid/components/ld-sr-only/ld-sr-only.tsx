import { Component, Host, h } from '@stencil/core'

@Component({
  tag: 'ld-sr-only',
  styleUrl: 'ld-sr-only.css',
  shadow: true,
})
export class LdSrOnly {
  render() {
    return (
      <Host class="ld-sr-only">
        <slot></slot>
      </Host>
    )
  }
}
