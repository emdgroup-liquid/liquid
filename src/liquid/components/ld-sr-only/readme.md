---
eleventyNavigation:
  key: Screen Reader Only
  parent: Components
layout: layout.njk
title: Screen Reader Only
permalink: components/ld-sr-only/
---

<link rel="stylesheet" href="{{ env.base }}/{{ buildstamp }}css_components/ld-sr-only.css">

# ld-sr-only

Use `ld-sr-only` to hide an element visually without hiding it from screen readers.

The CSS class `ld-sr-only` works the same way as its Web Component counterpart, except that it can be applied to an HTML element directly while the Web Component wraps the element which needs to be only screen reader visible.

---

## Examples

{% example '{ "opened": true }' %}
<ld-sr-only>Hello screen reader</ld-sr-only>

<!-- React component -->

<LdSrOnly>Hello screen reader</LdSrOnly>

<!-- CSS component -->

<span class="ld-sr-only">Hello screen reader</span>
{% endexample %}


<!-- Auto Generated Below -->


## Properties

| Property | Attribute | Description                                              | Type               | Default     |
| -------- | --------- | -------------------------------------------------------- | ------------------ | ----------- |
| `key`    | `key`     | for tracking the node's identity when working with lists | `string \| number` | `undefined` |
| `ref`    | `ref`     | reference to component                                   | `any`              | `undefined` |


## Dependencies

### Used by

 - [ld-file-upload](../ld-file-upload)
 - [ld-sidenav-header](../ld-sidenav/ld-sidenav-header)
 - [ld-sidenav-toggle-outside](../ld-sidenav/ld-sidenav-toggle-outside)
 - [ld-slider](../ld-slider)
 - [ld-sr-live](../ld-sr-live)
 - [ld-step](../ld-stepper/ld-step)
 - [ld-stepper](../ld-stepper)
 - [ld-tooltip](../ld-tooltip)
 - [ld-upload-item](../ld-file-upload/ld-upload-item)

### Graph
```mermaid
graph TD;
  ld-file-upload --> ld-sr-only
  ld-sidenav-header --> ld-sr-only
  ld-sidenav-toggle-outside --> ld-sr-only
  ld-slider --> ld-sr-only
  ld-sr-live --> ld-sr-only
  ld-step --> ld-sr-only
  ld-stepper --> ld-sr-only
  ld-tooltip --> ld-sr-only
  ld-upload-item --> ld-sr-only
  style ld-sr-only fill:#f9f,stroke:#333,stroke-width:4px
```

----------------------------------------------

*Built with [StencilJS](https://stenciljs.com/)*
