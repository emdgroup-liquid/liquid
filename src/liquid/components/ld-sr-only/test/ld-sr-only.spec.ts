import { newSpecPage } from '@stencil/core/testing'
import { LdSrOnly } from '../ld-sr-only'

describe('ld-sr-only', () => {
  it('renders', async () => {
    const page = await newSpecPage({
      components: [LdSrOnly],
      html: `<ld-sr-only>Hello screen reader</ld-sr-only>`,
    })
    expect(page.root).toMatchSnapshot()
  })
})
