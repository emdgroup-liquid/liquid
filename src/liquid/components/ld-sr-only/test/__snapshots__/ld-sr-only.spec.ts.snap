// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ld-sr-only renders 1`] = `
<ld-sr-only class="ld-sr-only">
  <mock:shadow-root>
    <slot></slot>
  </mock:shadow-root>
  Hello screen reader
</ld-sr-only>
`;
